src/monitors.spec.ts

```diff
@@ -237,6 +246,14 @@ const RECOMMENDED_MONITORS: RecommendedMonitors = {
     },
     templateVariables: TEMPLATE_VARIABLES,
   },
+  test_no_template_variable: {
```

Comment:
Should we have a test that calls `buildMonitorParams` with a `test_no_template_variable` monitor and verifies that the `name` is unchanged?

Reply:
Good point!